index.js

```diff
@@ -193,10 +193,11 @@ function fastifyView (fastify, opts, next) {
   }
 
   function viewEjsMate (page, data) {
-    if (!page || !data) {
+    if (!page) {
```

Comment:
I think the error message would need to be updated.

Reply:
Done.